src/lwc/geFormRenderer/elevateCaptureGroup.js

```diff
@@ -5,35 +5,28 @@ class ElevateCaptureGroup {
 
     constructor(elevateBatchId) {
```

Comment:
Passing the elevate batch Id into the constructor made sense when we wanted to try and group multiple gifts under the same batch. Now that we're moving to a 1:1 model between a gift and an elevate batch, I wonder if we should remove this?

Reply:
Updated!